docs/03-Usage.md

```diff
@@ -60,12 +60,11 @@ store:
 
 ##### ShowInSearch
 
-`ShowInSearch` is handled within the module. When it is set to false, the object is _removed_ from the Solr core.
+`ShowInSearch` is handled by the module itself, so there is no need to configure it within your YML/PHP index definition. When a content author sets this field to 0 via the CMS, then the related Page or File object is actually _removed_ from the applicable Solr core during the next run of the `SolrIndexJob` (which should be configured to run automatically).
```

Comment:
It shouldn't be done during the run of `SolrIndexJob`, it should be automatically done `onAfterPublish/onAfterWrite`, or if that fails, automatically through the `ClearDirtyClasses` task.

I understand those two methods are currently not doing what it should do exactly, but that's another fish to fry :D

Reply:
The issues mentioned are solved in #182